### PR TITLE
Update wizard to make request for domain info

### PR DESCRIPTION
**Pull request by huffman**

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/4613/HighFidelity-Beta-PR11729-8401aecef566c642c942e60f30975c186e4d5158.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/4613/HighFidelity-Beta-PR11729-8401aecef566c642c942e60f30975c186e4d5158.exe

**Comment by birarda** [comment]
#### Test Plan

https://highfidelity.testrail.net/index.php?/cases/view/2192

**Comment by cain-kilgore** [comment]
Testing.

**Comment by cain-kilgore** [comment]
# Test Failed
### Test Results on [TestRail Case 9198](https://highfidelity.testrail.net/index.php?/tests/view/9198&group_by=cases:section_id&group_order=asc&group_id=447)

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

**Comment by huffman** [comment]
@cain-kilgore Thanks for testing. Trying to figure out what went wrong. Did you copy the entire config? I'm wondering if the
```
"wizard": {
"cloud_domain": true
}
```
part didn't get in there, which would explain some of the failures.

**Comment by cain-kilgore** [comment]
@huffman It went in there as far as I could tell. If you want I can pass over the config I used.

**Comment by cain-kilgore** [comment]
# Test Passed
### Test Results on [TestRail Case 9198](https://highfidelity.testrail.net/index.php?/tests/view/9198&group_by=cases:section_id&group_order=asc&group_id=447)

Previous fail was the result of a bad config file.

###### Intel i5 7600K @ 4Ghz, 16GB DDR4 2133Mhz Corsair Vengeance, Asus Strix GTX 1050Ti 3GB, HTC Vive

File: domain-server/resources/web/js/shared.js [hunk 1]
```diff
@@ -146,187 +146,255 @@ function sendUpdatePlaceRequest(id, path, domainID, clearDomainID, onSuccess, on
...
-              className: 'add-place-cancel-button btn-default'
-            }
-          };
-
-          var dialog;
-          var modal_body;
+    function loadPlaces() {
+      $.ajax("/api/places", {
+        dataType: 'json',
+        jsonp: false,
```
Comment (birarda) [inline domain-server/resources/web/js/shared.js]: Not a blocker if it's the only comment - but this is implicit, right?

**Pull request merged**